types/wlr_output_layout.c

```diff
@@ -190,6 +229,45 @@ static struct wlr_output_layout_output *output_layout_output_create(
 	return l_output;
 }
 
+// Adapted from wl_list_for_each
+// Starts with wlr_ to avoid name conflicts
+#define wlr_wl_list_for_each_offset(pos, head, member, offset)			\
```

Comment:
I'd rather write a little less efficient but more readable code

Reply:
Rather, I'm not quite sure why you define this here. It's only used once until the #undef, so it can just be inline.

Reply:
That's a remnant from when I used it in two functions